app/views/accounts/_connected_app.html.erb

```diff
@@ -9,13 +9,17 @@
         <% end %>
       </div>
 
-      <%= t('account.connected_apps.associated',
-            timestamp: local_time(identity.created_at.in_time_zone('UTC'), t('time.formats.event_timestamp'))).html_safe %>
+      <%= t(
+            'account.connected_apps.associated',
+            timestamp: local_time(identity.created_at.in_time_zone('UTC'), t('time.formats.event_timestamp')),
+          ).html_safe %>
```

Comment:
I know it's not part of the lints so maybe we do this in a separate PR, but while we're here...

The translation for this is `Connected %{timestamp}` so I think we can remove the `.html_safe`

```suggestion
      <%= t(
            'account.connected_apps.associated',
            timestamp: local_time(identity.created_at.in_time_zone('UTC'), t('time.formats.event_timestamp')),
          ) %>
```

Reply:
Seems safe enough to include here 👍

Reply:
Not safe after all 😕 (LG-5472). `local_time` returns HTML markup.